delta-test/src/main/java/io/cdap/delta/test/DeltaPipelineTestBase.java

```diff
@@ -61,4 +68,13 @@ protected static void setupArtifacts(Class<?> appClass) throws Exception {
     addPluginArtifact(mocksArtifactId, ARTIFACT_ID, pluginClasses,
                       MockSource.class, MockTarget.class, MockErrorTarget.class, FailureTarget.class);
   }
+
+  protected static void enableCapability() throws IOException, InterruptedException {
+    String capabilityFolder = getConfiguration().get("capability.config.dir");
+    String cdcFile = "cdc.json";
+    URL url = DeltaPipelineTestBase.class.getClassLoader().getResource(cdcFile);
+    File outFile = new File(capabilityFolder, cdcFile);
+    ByteStreams.copy(Resources.newInputStreamSupplier(url), Files.newOutputStreamSupplier(outFile));
+    Thread.sleep(TimeUnit.MINUTES.toMillis(1));
```

Comment:
why do we need to add a sleep here? If we need to add this, can you add a comment inline for this?

Reply:
Created a bug for fixing this in a generic way. https://cdap.atlassian.net/browse/CDAP-17562